pipelines/binary-builder.yml

```diff
@@ -26,116 +27,101 @@ resources: #####################################################################
   - name: builds-out
     type: git
     source:
-      uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+      uri: git@github.com:SUSE/cf-public-buildpacks-ci-robots
       branch: master
       paths: [ binary-builds/* ]
       private_key: {{public-buildpacks-ci-robots-private-key}}
 
-<% dep_build_names = %w(bower go bundler composer dotnet glide godep httpd hwc jruby php php7 python node nginx ruby yarn) %>
 <% dep_build_names.each do |dep| %>
   - name: <%= dep %>-builds
     type: git
     source:
-      uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+      uri: git@github.com:SUSE/cf-public-buildpacks-ci-robots
       branch: master
       paths: [ binary-builds/<%= dep %>-builds.yml ]
       private_key: {{public-buildpacks-ci-robots-private-key}}
 <% end %>
 
-<% dep_built_output_names = dep_build_names + %w(dotnet-framework) %>
```

Comment:
The installer doesn't support openSUSE, it fails with a similar error to:
https://github.com/dotnet/cli/issues/2918

I think this would be enough effort to create a new ticket to get this fixed, and we can add the dotnet buildpack later.